testing/file_test/manifest.cpp

```diff
@@ -0,0 +1,21 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "testing/file_test/manifest.h"
+
+// The test manifest, produced by `manifest_as_cpp`.
+// NOLINTNEXTLINE(readability-identifier-naming): Constant in practice.
+extern const char* CarbonFileTestManifest[];
```

Comment:
The `extern` is unfortunate; using a symbol without #including the header that declares it can cause problems for tools that do things like cross-referencing and dependency management. Once we remove the Explorer, will we be able to shift to a model where the checked-in code #includes a generated header or something like that?

Reply:
Yes, this is basically jumping through extra hoops for explorer.

Reply:
Note, you can probably also switch around compiler techniques as-is. I was also kinda just taking an approach I was confident would work.